src/components/homeData.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="homeData">
+    <div v-bind:class="userInfo">
+      <p>{{currentUser.name}}さんようこそ！！</p>
+      <div v-bind:class="optionBox">
+        <p>残高：{{currentUser.balance}}</p>
+        <button v-bind:class="logoutBtn">ログアウト</button>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+  export default{
+    name: 'homeData',
+    data: function() {
+      return{
+        userInfo: 'user-info',
+        optionBox: 'option-box',
+        logoutBtn: 'logout-btn',
+      };
+    },
+    created: function() {
+      this.$store.dispatch('getUser');
+    },
+    methods: {
+     
+    },
+    computed: {
+      currentUser() {

```

Comment:
こちらは、以下の省略記法になっています。
```suggestion
      currentUser: function() {

```
なので、省略記法を使用するなら全てに統一すると良いです。